.vscode/settings.json

```diff
@@ -45,22 +45,21 @@
     "editor.formatOnSave": false
   },
 
-  "python.linting.pylintEnabled": false,
-  "python.linting.flake8Enabled": true,
+  "python.linting.pylintEnabled": true,
+  "python.linting.flake8Enabled": false,
```

Comment:
Why turn this off? I think we do use flake8

Reply:
Because vscode automatic stuff :D I'll fix it before landing.

>They can override workspace settings though can't they?

Not until this gets fixed: https://github.com/microsoft/vscode/issues/40233

Until then, the only option is change the file and then add it to your `.git/info/excludes` which is quite annoying.